fix: redesigned numbers section for assets and attendance

anirudhgovind opened the pull request.

anirudhgovind commented:
Addresses #10.

thedivtagguy commented:
Thanks for this. Some minor fixes I request. 

![image](https://github.com/thedivtagguy/whoismyneta/assets/6006888/98a6b127-c609-4fc6-8e01-7b752d37b6e1)

1. Add padding on top of relative wealth section to match the bottom section. 

2. Reorder the Assets/Criminal row. Criminal cases can come first, assets later (the assets cell should be aligned with the `Age` one above) so we don't have the placeholder text for `2024 assets` in the middle. 

3. Round off the attendance to an integer. Wanna try adding a tween to that bar too? You could wrap it with [this utility](https://svelte-ux.techniq.dev/docs/components/TweenedValue#style) from svelte-ux to make the transition between candidates nicer.

4. `text-2xl` for the values everywhere ('Graduate professional' etc.) can be reduced to `text-xl`.

5. Actually now I'm thinking if we're better off having a grid here instead of flex box so that all cells are left-aligned with each other on top and bottom. 

![image](https://github.com/thedivtagguy/whoismyneta/assets/6006888/75a24aa7-691b-4d32-96d9-821424306e96)

anirudhgovind commented:
Thanks for the feedback! Latest commit addresses these points. 

One thing I'd like help with is the font weight for some of the education values. It looks little different now that font size becomes smaller for longer values (Graduate Professional, etc). Quick fixes changing font-weight didn't look right.
